pkg/daprrp/frontend/controller/types.go

```diff
@@ -33,12 +33,12 @@ const (
 	// AsyncCreateOrUpdateDaprSecretStoreTimeout is the timeout for async create or update dapr secret store
 	AsyncCreateOrUpdateDaprSecretStoreTimeout = time.Duration(60) * time.Minute
 	// AsyncDeleteDaprSecretStoreTimeout is the timeout for async delete dapr secret store
-	AsyncDeleteDaprSecretStoreTimeout = time.Duration(30) * time.Minute
+	AsyncDeleteDaprSecretStoreTimeout = time.Duration(60) * time.Minute
```

Comment:
Was this always supposed to be 60 and just got missed?

Reply:
we were using "AsyncCreateOrUpdateDaprSecretStoreTimeout" variable for delete operation and which is 60 mins, so changed the value of "AsyncDeleteDaprSecretStoreTimeout" to keep the timeout value as it was before